R/standard_curves.R

```diff
@@ -115,7 +146,15 @@ plot_standard_curve_analyte <- function(plate, analyte_name,
 #' @import ggplot2
 #'
 #' @export
-plot_standard_curve_analyte_with_model <- function(plate, analyte_name, model, data_type = "Median", decreasing_dilution_order = TRUE, log_scale = c("all"), plot_asymptote = TRUE, verbose = TRUE) {
+plot_standard_curve_analyte_with_model <- function(plate,
+                                                   analyte_name,
+                                                   model,
+                                                   data_type = "Median",
+                                                   decreasing_dilution_order = TRUE,
+                                                   log_scale = c("all"),
+                                                   plot_asymptote = TRUE,
+                                                   plot_test_predictions = TRUE,
```

Comment:
I'd add some additional flags to control the plot behaviour, such as flag for adding/removing the blank mean, min-max dilution bounds, etc.